### ISSUE TITLE
Contents of XML code blocks gets erased

### ISSUE THREAD

**Issue by asdavey** [Laverna/laverna#378]
If while editing a note you add some xml inside of a code block and then hit save - the xml content is not displayed in the note window. Reopening the note to edit only reveals blank lines where the xml content was.

### Steps to Reproduce
1. Using the standalone desktop app start creating a new note
2. In the note content type something like:

    ```
         Below should be an xml fragment:
         ```xml
         <hello>world</hello>
         ```
         Above should say hello world in an xml fragment.
    ```
3. Note how you can preview the xml fragment.
4. Click save
5. View the note you just created.

### What happened next?
The note appears normally except for the contents of the code block - there is nothing there except blank lines.

It looked something like this:
> Below should be an xml fragment:
> ```xml
> ```
> Above should say hello world in an xml fragment.

### What I expected to happen?
I expected to see the note, with an xml fragment `<hello>world</hello>` displayed in the note.

### Additional Information
* Standalone app on OSX
* Encryption used

**Comment by wwwredfish** [comment]
Thanks for taking your time to report this issue. We will try to fix it as soon as possible.

**Comment by wwebfor** [comment]
The issue was fixed in 0.7.3-RC3. If you disagree, please let us know.
